android/src/com/mozilla/vpn/VPNServiceBinder.kt

```diff
@@ -246,7 +253,10 @@ class VPNServiceBinder(service: VPNService) : Binder() {
         ifaceBuilder.parsePrivateKey(privateKey)
         ifaceBuilder.addAddress(InetNetwork.parse(jDevice.getString("ipv4Address")))
         ifaceBuilder.addAddress(InetNetwork.parse(jDevice.getString("ipv6Address")))
-        
+
+        ifaceBuilder.addDnsServer(DNS.mullvadVpnInternal)
+        ifaceBuilder.addDnsServer(DNS.mullvadPublic);
```

Comment:
1 DNS is enough. We set it elsewhere too.
Btw, is this a bug in the existing android build too?

Reply:
no we do ` addDnsServer(InetNetwork.parse(serverInfo.server.ipv4Gateway).address)` there